src/aiperf/post_processors/otel_metrics_results_processor.py

```
@@ -0,0 +1,327 @@
+        for strategy in self._result_strategies:
+            if strategy.supports(record_data):
+                await strategy.process(record_data)
+                return
+
+        self.debug(
+            lambda: (
+                f"Skipping unsupported OTel result payload type: {type(record_data)}"
+            )
+        )
+
+    async def flush(self, *, force: bool = False) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `force`

(ARG002)